needrestart

```diff
@@ -157,6 +158,7 @@ our %nrconf = (
     containers => 1,
     },
     has_pam_systemd => 1,
+    tolerance => 0,
```

Comment:
Thinking again about this issue I would accept to change the default to `2` or maybe `1.01` ? I would not expect that non false-positives are happening that often (and the detection is just a heuristic).

Reply:
That's fine for me. I wasn't pushing for that, simply because I don't know other use cases.

I didn't re-check, but based on my original analysis, both the process start time and the file ctime have a precision of 1 second, so I think it would be best for the default tolerance to have the same precision.

Reply:
I made this change as well as changing some of the surrounding wording in the man page in order to better fit the new default.

Please forgive the multiple amendments; the branch should be in a good state now.

For the host I tested earlier:
```
# PERL5LIB=perl/lib ./needrestart -plk
OK - Kernel: <redacted>, Services: none, Containers: none, Sessions: none|Kernel=0;0;;0;2 Services=0;;0;0 Containers=0;;0;0 Sessions=0;0;;0
```